### PR TITLE
Add whitelist param to ITN

### PR DESCRIPTION
**Pull request by ekmb**
Signed-off-by: ekmb <ebakhturina@nvidia.com>

# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ X] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo_text_processing/inverse_text_normalization/inverse_normalize.py [hunk 1]
```diff
@@ -128,6 +131,13 @@ def parse_args():
     parser.add_argument(
         "--language", help="language", choices=['en', 'de', 'es', 'pt', 'ru', 'fr', 'vi'], default="en", type=str
     )
+    parser.add_argument(
+        "--whitelist",
+        help="Path to a file with with whitelist replacements,"
+        "e.g., for English whitelist files are stored under inverse_normalization/en/data/whitelist",
```
Comment (yzhang123) [inline nemo_text_processing/inverse_text_normalization/inverse_normalize.py:137]: should be normalization

File: nemo_text_processing/inverse_text_normalization/pt/taggers/whitelist.py [hunk 2]
```diff
@@ -22,12 +22,19 @@ class WhiteListFst(GraphFst):
     """
     Finite state transducer for classifying whitelisted tokens
         e.g. usted -> tokens { name: "ud." }
-    This class has highest priority among all classifier grammars. Whitelisted tokens are defined and loaded from "data/whitelist.tsv".
+    This class has highest priority among all classifier grammars.
+    Whitelisted tokens are defined and loaded from "data/whitelist.tsv".
```
Comment (yzhang123) [inline nemo_text_processing/inverse_text_normalization/pt/taggers/whitelist.py:26]: same here

File: nemo_text_processing/inverse_text_normalization/inverse_normalize.py [hunk 3]
```diff
@@ -28,6 +29,7 @@ class InverseNormalizer(Normalizer):
 
     Args:
         lang: language specifying the ITN
+        whitelist: path to a file with whitelist replacements
```
Comment (yzhang123) [inline nemo_text_processing/inverse_text_normalization/inverse_normalize.py:32]: could you also add the format of the whitelist, e.g. written tab normalized,

File: nemo_text_processing/inverse_text_normalization/de/taggers/whitelist.py [hunk 4]
```diff
@@ -24,11 +24,16 @@ class WhiteListFst(GraphFst):
         e.g. misses -> tokens { name: "Mrs." }
     Args:
         tn_whitelist_tagger: TN whitelist tagger
+        input_file: path to a file with whitelist replacements
```
Comment (yzhang123) [inline nemo_text_processing/inverse_text_normalization/de/taggers/whitelist.py:27]: could you add format of whitelist here, written\tnormalized

File: nemo_text_processing/inverse_text_normalization/inverse_normalize.py (hunk 1 above)
Comment (ekmb) [reply nemo_text_processing/inverse_text_normalization/inverse_normalize.py:137]: no, it's ITN

File: nemo_text_processing/inverse_text_normalization/de/taggers/whitelist.py (hunk 4 above)
Comment (ekmb) [reply nemo_text_processing/inverse_text_normalization/de/taggers/whitelist.py]: done

File: nemo_text_processing/inverse_text_normalization/inverse_normalize.py (hunk 3 above)
Comment (ekmb) [reply nemo_text_processing/inverse_text_normalization/inverse_normalize.py]: done

File: nemo_text_processing/inverse_text_normalization/pt/taggers/whitelist.py (hunk 2 above)
Comment (ekmb) [reply nemo_text_processing/inverse_text_normalization/pt/taggers/whitelist.py]: done

File: nemo_text_processing/inverse_text_normalization/inverse_normalize.py (hunk 1 above)
Comment (yzhang123) [reply nemo_text_processing/inverse_text_normalization/inverse_normalize.py]: yes but there is no whitelist file under inverse_text normalization//data

**Review by yzhang123 — APPROVED** [review]

**Pull request merged**